app/packages/tiling/src/views/MosaicGrid/MosaicGrid.module.css

```
@@ -52,18 +47,15 @@
   background: transparent;
 }
 
-/* Pull the per-tile margin in (react-mosaic defaults to 3px on each
-   side → 6px gap + 2px window borders ≈ 8px between tile contents).
-   2px gives 4px gap + 2px borders ≈ 6px — visibly slim without making
-   adjacent borders feel like one continuous line. */
+/* Blueprint-styled drag preview leaks through as a ghost — hide it. */
+.root :global(.mosaic .mosaic-preview) {
```

Stylelint (17.12.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)